shared/html/popup.html

```diff
@@ -17,6 +17,7 @@
 <!DOCTYPE html>
 <html>
 <head>
+    <meta charset="utf-8">
```

Comment:
@jdorweiler I added this line here because firefox keeps warning me about character encoding without it:

![image](https://user-images.githubusercontent.com/4481594/63390790-4a382580-c37e-11e9-9a6d-c1fd06316c22.png)

Reply:
Nice, thanks.